### DISCUSSION TITLE
Recent forum topics

### DISCUSSION THREAD

**Discussion by Jamiras** [RetroAchievements/RAWeb#727] [Ideas]
As a user, I don't want to have to look at multiple posts in a discussion I don't care about to get to other recent conversations I might have interest in.

The recent posts page (http://retroachievements.org/forumposthistory.php) currently lists all recent posts. I'd like a view that just shows recently active topics. Instead of five posts on one topic, two posts on another topic, a new topic, then three more posts on the first topic, I'd like to see a list that simply has first topic, second topic, third topic like you get when you're viewing a forum, but across all forums.

This is particularly important on the home page where only the last four posts are displayed. If one topic is dominating the home page, others may be overlooked. Clicking on the more link could go to an expanded list, and the forum activity could still show the most recent posts.

Note: "Forum Activity" currently appears on the home page (4 most recent), main forum page (http://retroachievements.org/forum.php) (8 most recent), and the forum post history (8 most recent), where it's completely redundant with the post history (25 most recent).

**Comment by Thoooreau** [comment]
This one is really necessary in Leapfrog times.
